Rename stocks to equity

piiq opened the pull request:
# Description

- [ ] Summary of the change / bug fix.
- [ ] Link # issue, if applicable.
- [ ] Screenshot of the feature or the bug before/after fix, if applicable.
- [ ] Relevant motivation and context.
- [ ] List any dependencies that are required for this change.


# How has this been tested?

* Please describe the tests that you ran to verify your changes.
* Provide instructions so we can reproduce.
* Please also list any relevant details for your test configuration.
- [ ] Make sure affected commands still run in terminal
- [ ] Ensure the SDK still works
- [ ] Check any related reports


# Checklist:

- [ ] I have adhered to the GitFlow naming convention and my branch name is in the format of `feature/feature-name` or `hotfix/hotfix-name`.
- [ ] Update [our documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).  Update any user guides that are affected by the changes.
- [ ] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [ ] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [ ] If a feature was added make sure to add it to the corresponding [integration test script](https://github.com/OpenBB-finance/OpenBBTerminal/tree/develop/openbb_terminal/miscellaneous/integration_tests_scripts).


# Others
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.

jmaslek commented on openbb_platform/extensions/equity/openbb_equity/estimates/estimates_router.py:
```diff
@@ -0,0 +1,48 @@
+
+
+@router.command(model="AnalystEstimates")
+def analyst(
```
should we be more verbose: analyst_estimates?

deeleeramone replied:
This function is probably better described by, `historical_earnings_consensus`

![Screenshot 2023-11-08 at 2 03 25 PM](https://github.com/OpenBB-finance/OpenBBTerminal/assets/85772166/d34a326e-74cb-413f-ae2e-754eed31d43b)

piiq replied:
how about `obb.equity.estimates.historical` here? then the other one can stay `obb.equity.estimates.consensus`

piiq replied:
i've put `obb.equity.estimates.historical` and `obb.equity.estimates.consensus`. We can re-think this in another PR

jmaslek commented on openbb_platform/extensions/equity/openbb_equity/estimates/estimates_router.py:
```diff
@@ -0,0 +1,48 @@
+
+
+@router.command(model="PriceTargetConsensus")
+def consensus(
```
Same here.  price_target_consensus ?

deeleeramone replied:
WDYT about, "analyst_consensus"?

piiq replied:
see comment above

jmaslek commented on openbb_platform/extensions/equity/openbb_equity/fundamental/fundamental_router.py:
```diff
@@ -93,18 +82,18 @@ def divs(
 
 @router.command(model="EarningsCalendar")
-def earning(
+def earnings(
```
Whats different here and the earnings calendar above?

deeleeramone replied:
This is not actually an earnings calendar.  This function actually returns,`historical_eps`

piiq replied:
still need a decision on this one. There is a bug on the models side where the model does not return what it should and yes there is a separate calendar `obb.equity.calendar.earnings` command

piiq replied:
IIUC this question will be addressed in this PR
https://github.com/OpenBB-finance/OpenBBTerminal/pull/5690

That PR will also need to address the renaming of the historical earnings model

jmaslek commented on openbb_platform/extensions/equity/openbb_equity/fundamental/fundamental_router.py:
```diff
@@ -93,18 +82,18 @@ def divs(
 
 @router.command(model="HistoricalEmployees")
-def emp(
+def employees(
```
maybe something like employee_count ?

jmaslek commented on openbb_platform/extensions/equity/openbb_equity/fundamental/fundamental_router.py:
```diff
@@ -279,19 +213,8 @@ def filings(
-
 @router.command(model="HistoricalStockSplits")
-def split(
+def splits(
```
same -- what is different from above?

deeleeramone replied:
There are `historical_splits`, by ticker, and then there should be a different calendar endpoint that does not go in the FA router.  The function `compsplit()` should be taken out of FA.

jmaslek commented on openbb_platform/extensions/equity/openbb_equity/shorts/shorts_router.py:
```diff
@@ -10,13 +10,13 @@
 
 @router.command(model="StockFTD")
-def ftd(
+def fail_to_deliver(
```
fails_to_deliver

deeleeramone commented on openbb_platform/extensions/equity/openbb_equity/calendar/calendar_router.py:
```diff
@@ -0,0 +1,59 @@
+    standard_params: StandardParams,
+    extra_params: ExtraParams,
+) -> OBBject[BaseModel]:
+    """Earnings Calendar. Earnings calendar for a given company."""
```
"""Upcoming and Historical earnings calendar."""

deeleeramone commented on openbb_platform/extensions/equity/openbb_equity/discovery/discovery_router.py:
```diff
@@ -11,7 +11,7 @@
 from pydantic import BaseModel
 
-router = Router(prefix="/disc")
+router = Router(prefix="/discovery")
```
`equity_discovery_undervalued_growth_equities` could trim`equities` from the end.

piiq replied:
![image](https://github.com/OpenBB-finance/OpenBBTerminal/assets/11668535/a30ace38-9e43-42a3-9498-76232ad2a080)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/OpenBB-finance/OpenBBTerminal?pullRequest=5688) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/OpenBB-finance/OpenBBTerminal?pullRequest=5688) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/OpenBB-finance/OpenBBTerminal?pullRequest=5688) it.</sub>

piiq commented:
So i've fixed a few bugs and updated the tests. I've uncovered failing integration tests on the web api side in the discovery menu. Also the comments that have not been resolved are not resolved. @IgorWounds please give some feedback (see discussions above) and it will be then ready to merge

piiq commented:
not exactly. the tests that rely on the tests fail. fixing

piiq commented:
it's good from my side. i've identified a few flaky api tests, but they should not blocking this merge

deeleeramone commented:
> it's good from my side. i've identified a few flaky api tests, but they should not blocking this merge

So, when an item, like a symbol directory, the cassettes will not record a cached request event.  If functions share a cached resource, it will only capture the cassette for the first instance.  The workaround is to delete the cache file and then only run the single test which needs to be recorded.

I've captured the missing cassettes and commented out the `skip`.
